js/modules/k6/http/request_test.go

```diff
@@ -1348,6 +1348,11 @@ func TestErrorCodes(t *testing.T) {
 	defer tb.Cleanup()
 	sr := tb.Replacer.Replace
 
+	var connectionRefusedErrorText = "connect: connection refused"
+	if runtime.GOOS == "windows" {
+		connectionRefusedErrorText = "connectex: No connection could be made because the target machine actively refused it."
```

Comment:
Why is the "No" captialized but the "connection" (above) not?

Reply:
This is what windows/golang gives us back ... and as far as I can find out this windows text comes from outside go ... so :(

Reply:
Ah, missed that it was in a test case actually, was in the mindset that it was a string we set for `res.error` 😄